ui/page/root/home_page.go

```diff
@@ -151,7 +151,6 @@ func (hp *HomePage) OnNavigatedTo() {
 		}
 	}
 
-	hp.CurrentPage().OnNavigatedTo()
```

Comment:
looks like this caused a crash else where. when this line is included, we get a panic at 

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x134e042]

goroutine 45 [running]:
github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).listenForMixerNotifications.func1()
        /home/cpunch/projects/cryptopower/ui/page/root/overview_page.go:878 +0x2c2
created by github.com/crypto-power/cryptopower/ui/page/root.(*OverviewPage).listenForMixerNotifications in goroutine 37
        /home/cpunch/projects/cryptopower/ui/page/root/overview_page.go:848 +0x6f8
```